taqueria-vscode-extension/src/lib/gui/SandboxesDataProvider.ts

```diff
@@ -156,22 +160,26 @@ export class SandboxesDataProvider extends TaqueriaDataProviderBase
 		const address = accountItem instanceof SandboxImplicitAccountTreeItem
 			? accountItem.address
 			: accountItem.parent.address;
-		const tzktBaseUrl = this.sandboxStates[accountItem.parent.sandboxName]?.getTzKtBaseUrl(
-			accountItem.parent.sandboxName,
-		);
+		const tzktBaseUrl = await this.getTzKtBaseUrl(accountItem.parent.sandboxName);
 		if (!tzktBaseUrl) {
 			return [];
 		}
-		const response = await fetch(`${tzktBaseUrl}/v1/accounts/${address}/operations`);
-		const data = await response.json();
-		return (data as any[]).map(item =>
-			new OperationTreeItem(
-				item.type,
-				item.hash,
-				item,
-				accountItem instanceof SandboxImplicitAccountTreeItem ? accountItem : accountItem.parent,
-			)
-		);
+		try {
+			// TODO: query filter - only operations from config.json ???
+			const response = await fetch(`${tzktBaseUrl}/v1/accounts/${address}/operations`);
+			const data = await response.json();
+			return (data as any[]).map(item =>
+				new OperationTreeItem(
+					item.type,
+					item.hash,
+					item,
+					accountItem instanceof SandboxImplicitAccountTreeItem ? accountItem : accountItem.parent,
+				)
+			);
+		} catch (e) {
+			this.helper.logAllNestedErrors(e, true);
+			return [];
```

Comment:
Returning `[]` silently, can mislead the user to believe there are no operations. One solution can be to not pass `suppressWindow: true` to `logAllNestedErrors()`, so that the user knows a request has failed. Another would be to have an `ErrorTreeItem` and return an array of one of those, which is displayed with a red icon and some error text.

Reply:
Actually, that was your approach with `supressWindow `🤔 

![image](https://user-images.githubusercontent.com/10618781/209371714-b2b320b4-ee85-4c39-afc9-61ad9d4b9498.png)

Reply:
So we should pass `true` only if sandbox is not running yet?

Reply:
IIRC in those we return `undefined`, so that the GUI will show nothing (not for instance 0), so the user knows that something is not working. If instead, we show no children under operations, the user can infer that there are actually no operations. So I think showing a visual indication of error is better.